### PR TITLE
add wackxu as reviewer

### PR DESCRIPTION
**Pull request by wackxu**
Signed-off-by: wackxu <xushiwei5@huawei.com>

**What type of PR is this?**

PRs authored:

https://github.com/kubeedge/kubeedge/pulls/wackxu

reviews:
https://github.com/kubeedge/kubeedge/pulls?q=is%3Apr+reviewed-by%3Awackxu

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
/assign @kevin-wangzefeng

File: tests/e2e/OWNERS [hunk 1]
```diff
@@ -3,8 +3,10 @@ approvers:
   - kadisi
   - kevin-wangzefeng
   - sids-b
+  - wackxu
 reviewers:
   - anyushun
   - fisherxu
   - kadisi
   - kuramal
+  - wackxu
```
Comment (kevin-wangzefeng) [inline tests/e2e/OWNERS:12]: No newline at end of file. Pleae fix the format issue.

**Review by kevin-wangzefeng — COMMENTED** [review]
@wackxu has made numerous contributions to the project, +1 to the membership promotion.
/approve

**Comment by gy95** [comment]
/lgtm
cc @fisherxu

**Pull request merged**